cogs/deletelast/deletelast.py

```diff
@@ -0,0 +1,28 @@
+import logging
+
+import discord
+from discord.ext import commands
+from bot import Zhenpai
+
+log: logging.Logger = logging.getLogger(__name__)
+
+class DeleteLast(commands.Cog):
+    """ Purge the last N messages to repent sins. """
+
+    def __init__(self, bot: Zhenpai):
+        self.bot = bot
+
+    def cog_unload(self):
+        self.check_reminders.cancel()
```

Comment:
I don't actually know what this does tbh

Reply:
oh you dont need this it actually prob errors, did you copy this from another modules, iirc it's cus the reminders cog has/had a background task and when the cog unloads you want to kill the task